designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputDropDown.kt

```diff
@@ -76,22 +79,27 @@ fun InputDropDown(
     title: String,
     state: InputShellState,
     inputStyle: InputStyle = InputStyle.DataInputStyle(),
-    dropdownItems: List<DropdownItem>,
+    itemCount: Int,
+    onSearchOption: (String) -> Unit,
+    fetchItem: (index: Int) -> DropdownItem,
```

Comment:
please, can you update the documentation for InputDropdown component with the new parameters?